runtime/minor_gc.c

```diff
@@ -410,10 +420,14 @@ void caml_empty_minor_heap (void)
     Caml_state->extra_heap_resources_minor = 0;
     caml_gc_message (0x02, ">");
     Caml_state->in_minor_collection = 0;
+    caml_ev_begin(EV_MINOR_FINALIZED);
     caml_final_empty_young ();
     CAML_INSTR_TIME (tmr, "minor/finalized");
+    caml_ev_end(EV_MINOR_FINALIZED);
```

Comment:
`caml_final_empty_young` returns immediately, it does not finalises by itself. Is this your intended semantics of the scope EV_MINOR_FINALIZED? I would have expected a punctual event.

Reply:
Looks like an oversight, thank you. I expended the event to match the intended semantic.
I think a punctual event could work as well, but it sounds like it's overlapping with the notion of `entry`/`exit` events in this case, but I may be wrong?
This event could likely even be left out as the end of the encompassing event slice/start of the next slice would acts as the same marker. (in the intended semantic of this case in particular).
I'm undecided on the better approach but if you feel like this is not optimal I can adjust it, adding punctual events is rather easy but would maybe require reconsidering a few other, similar cases, in what is traced for consistency's sake.

Reply:
Looks like an oversight, thank you. I expended the event to match the intended semantic.
I think a punctual event could work as well, but it sounds like it's overlapping with the notion of `entry`/`exit` events in this case, but I may be wrong?
This event could likely even be left out as the end of the encompassing event slice/start of the next slice would acts as the same marker. (in the intended semantic of this case in particular).
I'm undecided on the better approach but if you feel like this is not optimal I can adjust it, adding punctual events is rather easy but would maybe require reconsidering a few other, similar cases, in what is traced for consistency's sake.

Reply:
I do not have opinions on the semantics (I am not sure what it is), I just wanted to raise a possible oversight. Feel free to mark it as solved if you are satisfied with your solution.